flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -792,6 +792,20 @@ object AggregateUtil {
         inputType,
         needRetraction = false)
 
+    val forwardMapping = (0 until inputType.getFieldCount).map(x => (x, x)).toArray
```

Comment:
If we would call the `setForwardFields()` method, we would override something.
We should pass an empty array such that the method doesn't do anything.